packages/server/src/instances/service.ts

```diff
@@ -80,59 +71,41 @@ export class InstanceService extends Service {
   }
 
   async setup() {
-    this.lazyLoadingEnabled = !yn(process.env.NO_LAZY_LOADING)
-
-    this.cache = await this.cachingService.newServerCache('cache_instance_by_conduit_id', {
-      dispose: async (k, v) => {
-        if (!this.destroyed) {
-          await this.handleCacheDispose(k, v)
-        }
-      },
-      max: 50000,
-      maxAge: ms('30min')
-    })
```

Comment:
We'll need to put back this logic of clearing channels after not being used for x amount of minutes. To be done in future PR.